src/plus/startWork/startWork.ts

```diff
@@ -569,3 +588,15 @@ function isStartWorkTypeItem(item: unknown): item is StartWorkTypeItem {
 export function getStartWorkItemIdHash(item: StartWorkItem) {
 	return md5(item.item.issue.id);
 }
+
+function getOpenOnWebQuickInputButton(integrationId: string): QuickInputButton | undefined {
+	switch (integrationId) {
+		case HostingIntegrationId.GitHub:
+		case SelfHostedIntegrationId.GitHubEnterprise:
+			return OpenOnGitHubQuickInputButton;
```

Comment:
Do we support GitHub enterprise? Will this interface show you issues from your GitHub Enterprise instance if you have it connected? If not, we shouldn't include it here yet.

Reply:
fixed